entgql/internal/todo/ent.resolvers.go

```diff
@@ -53,11 +54,43 @@ func (r *queryResolver) Users(ctx context.Context, after *ent.Cursor, first *int
 		)
 }
 
+func (r *todoResolver) Init(ctx context.Context, obj *ent.Todo) (map[string]interface{}, error) {
```

Comment:
You can add `Map` [here](https://github.com/ent/contrib/blob/master/entgql/internal/todo/gqlgen.yml#L23) to avoid these mappers: 
```diff
models:
  ID:
    model:
      - github.com/99designs/gqlgen/graphql.IntID
  Uint64:
    model:
      - github.com/99designs/gqlgen/graphql.Uint64
+  Map:
+    model:
+      - github.com/99designs/gqlgen/graphql.Map
```